src/libdrakvuf/win.h

```diff
@@ -110,6 +110,10 @@
 #include "os.h"
 #include "win-exports.h"
 
+#define POOL_TAG_VADL (0x6c646156)
+#define POOL_TAG_VAD (0x20646156)
+#define POOL_TAG_VADM (0x20646156)
+
```

Comment:
Perhaps move these definitions into win-processes.c, that's the only spot they are needed